### PR TITLE
Website

### PR DESCRIPTION
**Pull request by SpeediBoi**

### CODE REVIEW

File: sub-logs/propkit.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "repo": "https://github.com/SpeediBoi/register",
+
+  "owner": {
+    "username": "SpeediBoi",
+    "email": "@Tejas6771@"
+  },
+
+  "target": {
+    "record-type": {
```
Comment (WilliamDavidHarrison) [inline sub-logs/propkit.json:10]:
```suggestion
    "CNAME": {
```

File: sub-logs/propkit.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "repo": "https://github.com/SpeediBoi/register",
+
+  "owner": {
+    "username": "SpeediBoi",
+    "email": "@Tejas6771@"
```
Comment (WilliamDavidHarrison) [inline sub-logs/propkit.json:6]: Please replace with a valid email

File: sub-logs/propkit.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "repo": "https://github.com/SpeediBoi/register",
```
Comment (WilliamDavidHarrison) [inline sub-logs/propkit.json:2]:
```suggestion
```

Invalid

File: sub-logs/propkit.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
```
Comment (WilliamDavidHarrison) [inline sub-logs/propkit.json:3]:
```suggestion
```

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

**Comment by WilliamDavidHarrison** [comment]
Deleted empty file ad85f9492a9a97af547f1e3192206c01948c299a

**Comment by Tweak4141** [comment]
@SpeediBoi please fix.

**Pull request closed without merging**